man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
```

Comment:
maybe drop `Network` from here? it's kinda obvious

Reply:
That makes more clear definition. why short name ?

Reply:
I don't think saying "network" adds anything here... it's a network configuration file, under a "QueueingDiscipline" section... what else could the delay refer to?
and really I could ask you the same question about many existing config keys, why is it `Address=` and not `NetworkAddress=`? `Gateway=` and not `NetworkGateway=`? So on so forth.

Reply:
I think network actually enhances the readability and you can't compare with gateway here

Reply:
What do you mean by heh ?

Reply:
> What do you mean by heh ? Nice that you figured out now. Another comment when after I explained, How do I know you know I mean you think you know or later you know

Reply:
What is mean is that I always knew this commit was about the netem module (with which I am very familiar), however I did not realize that your abbreviation "Network" was supposed to refer to "NetworkEmulator".  That is completely unintuitive IMHO. Anyway, I saw you changed to `NetworkEmulator` in the newest patch, so I'll accept that (although I'm still not 100% happy with it).

Reply:
Yeah I should have done that but in code I already  there 'network_emulator`